### ISSUE TITLE
unable to set default friction on static body

### ISSUE THREAD

**Issue by verneroberts** [liabru/matter-js#645]
when creating a body with options { isStatic:true, friction:0.01 } the friction is set to 1 instead of 0.01
I've traced this down to the setStatic method line 265, part.friction = 1;

**Comment by jcyuan** [comment]
yes agree, this should be a bug, static objects still have friction, so I think you couldn't reset friction to 1 in Body.setStatic method? @liabru
